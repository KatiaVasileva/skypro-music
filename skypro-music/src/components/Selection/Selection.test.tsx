import ReduxProvider from "@/store/ReduxProvider";
import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import SelectionItem from "./Selection";

describe("Selection component", () => {
  it("renders correctly", () => {
    const { container } = render(
      <ReduxProvider>
        <SelectionItem id={""} />
      </ReduxProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
