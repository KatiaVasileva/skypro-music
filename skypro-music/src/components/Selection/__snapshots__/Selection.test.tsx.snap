// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selection component renders correctly 1`] = `
<div>
  <div
    class="item"
  >
    <a
      href="/playlist/selection"
    >
      <img
        alt="day's playlist"
        class="img"
        data-nimg="1"
        decoding="async"
        fetchpriority="high"
        height="150"
        src="/_next/image?url=%2Fimg%2Fplaylist0.png&w=640&q=75"
        srcset="/_next/image?url=%2Fimg%2Fplaylist0.png&w=256&q=75 1x, /_next/image?url=%2Fimg%2Fplaylist0.png&w=640&q=75 2x"
        style="color: transparent;"
        width="250"
      />
    </a>
  </div>
</div>
`;
